packages/google_maps_flutter/google_maps_flutter/android/src/test/java/io/flutter/plugins/googlemaps/MarkersControllerTest.java

```diff
@@ -0,0 +1,131 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.googlemaps;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import com.google.android.gms.internal.maps.zzt;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import io.flutter.plugin.common.BinaryMessenger;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodCodec;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class MarkersControllerTest {
+
+  @Test
+  public void controller_OnMarkerDragStart() {
+    final MethodChannel methodChannel =
+        spy(new MethodChannel(mock(BinaryMessenger.class), "no-name", mock(MethodCodec.class)));
+    final MarkersController controller = new MarkersController(methodChannel);
+    final GoogleMap googleMap = mock(GoogleMap.class);
+    controller.setGoogleMap(googleMap);
+
+    final zzt z = mock(zzt.class);
```

Comment:
This may actually be very fragile if it's actually obfuscated code (with a name like that it could very well be).
I think that zzt might be something to do with the map projection but don't quote me, it's been a year and a half since I wrote that so I'm not totally sure without digging back into the source code as creation of Markers is hidden behind a factory pattern.
I just tested mocking only Marker and the test work fine, I'm not not sure why I didn't do that in the first place.
I'll create a new PR with this change.

Reply:
#4400

Reply:
Thanks for coming back to this one, I'll take a look at the new PR!!